docker/join_layers.py

```diff
@@ -87,13 +90,24 @@ def main():
 
   tag_to_layer = {}
   layer_to_tags = {}
+  stamp_info = {}
+
+  if args.stamp_info_file:
+    for infofile in args.stamp_info_file:
+      with open(infofile) as info:
+        for line in info:
+          line = line.strip("\n")
+          key, value = line.split(" ")
```

Comment:
you probably want `line.split(" ", 1)` (or maybe 2, I can never remember the sense of this)